companies/1und1-webde.json

```diff
@@ -26,7 +26,7 @@
         "WEB.DE Internet Fax",
         "WEB.DE Magazin"
     ],
-    "address": "c/o 1&1 Mail & Media GmbH\nElgendorfer Straße 57\n56410 Montabaur\nDeutschland",
+    "address": "c/o 1&1 Mail & Media GmbH\nElgendorfer Straße 57\n56410 Berlin",
```

Comment:
🚫 **[recordlint]** <[end-address-in-country](https://github.com/datenanfragen/data#addresses)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Last line of `address` (`56410 Berlin`) is not a country.

We've decided on specific variations for these countries:
* `United States of America`
* `The Netherlands`
* `Republic of Singapore`
```suggestion
    "address": "c/o 1&1 Mail & Media GmbH\nElgendorfer Straße 57\n56410 Berlin\nGermany",
```
```suggestion
    "address": "c/o 1&1 Mail & Media GmbH\nElgendorfer Straße 57\n56410 Berlin\nUnited States of America",
```